Gemfile

```diff
@@ -45,6 +45,7 @@ end
 
 group :test do
   gem "ci_reporter"
+  gem "launchy"
```

Comment:
It could, but I couldn't have completed this commit without it as I used `save_and_open_page` a lot to debug the spec. So as the first place where it's presence was required seems the right place for it to be added.

Reply:
Agreed useful change - but will have as separate commit.